ops/staging-deploy.tmpl.yaml

```diff
@@ -12,6 +12,8 @@ derivativesVolume:
 uploadsVolume:
   storageClass: aws-efs
   size: 200Gi
+logs:
+  storageClass: efs-sc
```

Comment:
@orangewolf bringing extra attention to this addition

Reply:
you don't have enough parts here I don't think. you'd need to do something to the hyrax chart to make `logs:` a valid option. as a less cumbersome alternative, see the suggestion above